app/assets/javascripts/components/activity/upload_table.jsx

```diff
@@ -25,46 +20,20 @@ const UploadTable = createReactClass({
     });
   },
 
-  _renderHeaders() {
-    return this.props.headers.map((header) => {
-      if (header.key !== 'image') {
-        return (
-          <th key={header.key} style={header.style || {}} onClick={this.sortItems} className="sortable asc" data-sort-key={header.key}>
-            {header.title}
-            <span className="sortable-indicator" />
-          </th>
-        );
-      }
-      return (
-        <th key={header.key} style={header.style || {}} className="sortable">
-          {header.title}
-        </th>
-      );
-    });
-  },
-
   render() {
     if (this.props.loading) {
       return <Loading />;
     }
 
     const uploads = this._renderUploads();
-    const ths = this._renderHeaders();
 
     return (
-      <table className="uploads table table--sortable">
-        <thead>
-          <tr>
-            {ths}
-            <th />
-          </tr>
-        </thead>
-        <tbody>
-          {uploads}
-        </tbody>
-      </table>
+      <div className="gallery">
+        {uploads}
+      </div>
```

Comment:
I think you could replace the usage of `UploadTable` altogether, using `UploadList` in `RecentUploadsHandler instead of this component.